### ISSUE TITLE
api.idaapi.FlowChart error

### ISSUE THREAD

**Issue by jinyu00** [williballenthin/python-idb#40]
I want to get all basic block in a function. so I write the below code
```
#!/usr/local/bin/python
# -*- coding:utf-8 -*-
import idb

idb_path = "/home/haclh/sec_tools/fuzz_utils/python-idb/tests/data/v7.0b/x64/kernel32.i64"
with idb.from_file(idb_path) as db:
    root = idb.analysis.Root(db)
    api = idb.IDAPython(db)

    fva = 0x6890615B
    func = api.ida_funcs.get_func(fva)
    bbs = list(api.idaapi.FlowChart(func))
    f = api.idaapi.FlowChart(func)
    for bb in f:
        print("Basic Block strat addr： ".format(bb.startEA))

```
but have an error

```
ssh://haclh@192.168.245.130:22/usr/bin/python -u /home/haclh/sec_tools/fuzz_utils/python_idb_learn.py
No handlers could be found for logger "idb.fileformat"
Traceback (most recent call last):
  File "/home/haclh/sec_tools/fuzz_utils/python_idb_learn.py", line 12, in <module>
    bbs = list(api.idaapi.FlowChart(func))
  File "build/bdist.linux-x86_64/egg/idb/idapython.py", line 1655, in FlowChart
  File "build/bdist.linux-x86_64/egg/idb/idapython.py", line 1593, in __init__
  File "build/bdist.linux-x86_64/egg/idb/idapython.py", line 1476, in _find_bb_end
  File "build/bdist.linux-x86_64/egg/idb/idapython.py", line 617, in NextHead
AttributeError: ida_bytes instance has no attribute 'isHead'

Process finished with exit code 1

```

**Comment by williballenthin** [comment]
i found recently that i had named some methods incorrectly (based on how Hex-Rays does it), so i updated them. looks like i missed some references. i'll get this patched up soon.

thanks for the report!

**Comment by williballenthin** [comment]
fixed in 9e5e3ba1baa8125c596b5afab6a5f55790c372fa
